src/reactpy/core/vdom.py

```diff
@@ -135,6 +135,13 @@ def __init__(
             self.__module__ = module_name
             self.__qualname__ = f"{module_name}.{tag_name}"
 
+    def __getattr__(self, attr: str) -> Vdom:
+        return Vdom(
+            f"{self.__name__}.{attr}",
+            allow_children=self.allow_children,
+            import_source=self.import_source,
+        )
+
```

Comment:
This method should throw an error if `self.import_source is None`, since this method can't be used alongside `reactpy.html` elements.

Reply:
Done. And I wrote a test for it.